### ISSUE TITLE
Elm Application Crashes ... using a ScreenReader 

### ISSUE THREAD

**Issue by andre-dietrich** [elm/browser#121]
I just found out that the application of ScreenReaders for blind, such as the Chrome ScreenReaders results in a crash of an elm application. It seems to add additional nodes into the application, that elm cannot handle.

Tested with: https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn

![grafik](https://user-images.githubusercontent.com/3089101/108722645-ac4d8a00-7523-11eb-969a-7a6c7449b49d.png)

It seems, that this problem can be fixed by applying the following patch:

https://github.com/jinjor/elm-break-dom

Probably this could be integrated in general, since it seems to solve a couple of other problems with browser extensions.
